libraries/core/Cargo.toml

```diff
@@ -22,3 +22,4 @@ tokio = { version = "1.24.1", features = ["fs", "process", "sync"] }
 aligned-vec = { version = "0.5.0", features = ["serde"] }
 schemars = "0.8.19"
 serde_json = "1.0.117"
+communication-layer-request-reply = { workspace = true }
```

Comment:
I would prefer if we kept the communication itself out of the `core` library. The individual components use different communication patterns, so it's probably better to keep them separated.

Reply:
I reverted this change